backend/src/litigation_api/ingestion/client.py

```
@@ -0,0 +1,109 @@
+            wait=wait_exponential(multiplier=1, min=2, max=10),
+            retry=retry_if_exception_type((httpx.RequestError, httpx.HTTPStatusError, CRLCAClientError)),
+            reraise=True
+        )
+        async def _download() -> bytes:
+            try:
+                # We use the existing client; if url is absolute, httpx handles it correctly
+                response = await self.client.get(url)
+                response.raise_for_status()
+                return response.read()
+            except httpx.HTTPStatusError as e:
+                raise CRLCAClientError(f"HTTP error downloading {url}: {e}") from e
```

Ruff (0.15.11)

[warning] Avoid specifying long messages outside the exception class

(TRY003)